### PR TITLE
Doc Fixes

### PR DESCRIPTION
**Pull request by msaroufim**
* Image was very blurry
* Markdown formatting was off
* Simplified some sentences

### CODE REVIEW

**Review by gnadathur — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -1,63 +1,65 @@
 # torchtitan
 
-## torchtitan is still in pre-release!
 `torchtitan` is currently in a pre-release state and under extensive development.
 
 `torchtitan` is a proof-of-concept for Large-scale LLM training using native PyTorch. It is (and will continue to be) a repo to showcase PyTorch's latest distributed training features in a clean, minimal codebase. torchtitan is complementary to and not a replacement for any of the great large-scale LLM training codebases such as Megatron, Megablocks, LLM Foundry, Deepspeed, etc. Instead, we hope that the features showcased in torchtitan will be adopted by these codebases quickly. torchtitan is unlikely to ever grow a large community around it.
 
-Here are some principles around building `torchtitan`
+Here were our guiding principles when building `torchtitan`
```
Comment (wanchaol) [inline README.md]: nit: we need a trailing `:`?

File: README.md [hunk 2]
```diff
@@ -1,63 +1,65 @@
...
 
-[![Welcome to TorchTrain!](assets/images/titan_play_video.jpg)](https://youtu.be/ee5DOEqD35I?si=_B94PbVv0V5ZnNKE "Welcome to TorchTrain!")
+[![Welcome to torchtitan!](assets/images/titan_play_video.png)](https://youtu.be/ee5DOEqD35I?si=_B94PbVv0V5ZnNKE "Welcome to torchtitan!")
 
 ## Pre-Release Updates:
 #### (4/25/2024): `torchtitan` is now public but in a pre-release state and under development.
-Currently we showcase pre-training **Llama 3 and Llama 2** models (LLMs) of various sizes from scratch. `torchtitan` is tested and verified with the PyTorch nightly version `torch-2.4.0.dev20240412`. (We recommend latest PyTorch nightly).
+Currently we showcase pre-training **Llama 3 and Llama 2** LLMs of various sizes from scratch. `torchtitan` is tested and verified with the PyTorch nightly version `torch-2.4.0.dev20240412`. (We recommend latest PyTorch nightly).
 
 Key features available:</br>
```
Comment (wanchaol) [inline README.md]: shall we remove this `</br>` too? (might need to indent one line for markdown to render correctly

**Review by malfet — APPROVED** [review]

**Review by wanchaol — APPROVED** [review]
This looks awesome and much better than before!! Thanks for the updates!

**Pull request merged**